templates/bridgeserver2.yaml

```diff
@@ -0,0 +1,680 @@
+Description: Resources for BridgeServer2 Application
+AWSTemplateFormatVersion: 2010-09-09
+Parameters:
+  AppDeployBucket:
+    Type: String
+  AppHealthcheckUrl:
+    Type: String
+    Description: The AWS EB health check path
+  AwsAutoScalingMaxSize:
+    Type: String
+    Default: '1'
+  AwsAutoScalingMinSize:
+    Type: String
+    Default: '1'
+  AwsVpcId:
+    Type: "AWS::EC2::VPC::Id"
+    Description: The AWS account's default VPC id
+  AwsEbHealthReportingSystem:
+    Type: String
+    Default: basic
+    AllowedValues:
+      - basic
+      - enhanced
+    ConstraintDescription: must be either basic or enhanced
+  AwsSnsBounceNotificationEndpoint:
+    Type: String
+    Description: Email address for SNS bounce notifications
+  AwsSnsNotificationEndpoint:
+    Type: String
+    Description: Email address for AWS SNS notifications
+  AwsSolutionStackName:
+    Description: The AWS Solution Stack
+    Type: String
+  BridgeEnv:
+    Type: String
+    Default: dev
+    AllowedValues:
+      - prod
+      - uat
+      - dev
+    ConstraintDescription: must specify dev, uat or prod.
+  BridgeUser:
+    Type: String
+    Default: heroku
+  DNSDomain:
+    Type: String
+    Description: DNS Domain name for deployment
+    Default: sagebridge.org
+  DNSHostname:
+    Type: String
+    Description: DNS Hostname for deployment
+  EC2InstanceType:
+    Type: String
+    Description: Instance type to use for Elastic Beanstalk Instances
+    Default: t2.small
+    AllowedValues:
+      - t2.nano
+      - t2.micro
+      - t2.small
+      - t2.medium
+      - t2.large
+      - t2.xlarge
+      - t2.2xlarge
+  Env:
+    Type: String
+    Default: develop
+    AllowedValues:
+      - prod
+      - staging
+      - develop
+Conditions:
+  CreateDevResources: !Equals [ !Ref BridgeEnv, dev ]
+Resources:
+  AWSEBConfigurationTemplate:
+    Type: 'AWS::ElasticBeanstalk::ConfigurationTemplate'
+    Properties:
+      ApplicationName: !ImportValue us-east-1-bridgeserver2-common-BeanstalkAppName
+      SolutionStackName: !Ref AwsSolutionStackName
+      OptionSettings:
+        # EB environment options
```

Comment:
you don't have an application load balancer configuration.  if you don't configure an ALB i believe beanstalk will setup classic load balancer for you by default.  You probably don't want that because it's not easy to switch from classic to application LB.  I don't think cloudformation can do that update.

Reply:
to setup an ALB you need to configure `aws:elbv2:loadbalancer`[1] in this ConfigurationTemplate section.  Try checking agora template again[2], it's in there. 

[1] https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/command-options-general.html#command-options-general-elbv2
[2] https://github.com/Sage-Bionetworks/Agora-infra/blob/develop/templates/beanstalk.yaml

Reply:
i'm not sure, but i think you might also need to set `aws:elbv2:listener:default`.